### PR TITLE
[cherry-pick] Bulkinsert read varchar by batch

### PR DESCRIPTION
**Pull request by yhmo**
#26197

### CODE REVIEW

**Comment by yhmo** [comment]
rerun ut

**Comment by yhmo** [comment]
rerun ut

File: internal/util/importutil/numpy_adapter.go [hunk 1]
```diff
@@ -572,36 +589,54 @@ func (n *NumpyAdapter) ReadString(count int) ([]string, error) {
 			// the character "a" occupys 2*4=8 bytes(0x97,0x00,0x00,0x00,0x00,0x00,0x00,0x00),
 			// the "bb" occupys 8 bytes(0x97,0x00,0x00,0x00,0x98,0x00,0x00,0x00)
 			// for non-ascii characters, the unicode could be 1 ~ 4 bytes, each character occupys 4 bytes, too
-			raw, err := ioutil.ReadAll(io.LimitReader(n.reader, utf8.UTFMax*int64(maxLen)))
+			raw, err := ioutil.ReadAll(io.LimitReader(n.reader, utf8.UTFMax*int64(maxLen)*int64(batchRead)))
```
Comment (czs007) [inline internal/util/importutil/numpy_adapter.go:592]: Add more unit test?

**Pull request merged**